### PR TITLE
DAOS-10651 test: Install daos-tests-internal for weekly tests

### PR DESCRIPTION
**Pull request by phender**
The daos-tests-clients-openmpi RPM is now installed via the
daos-tests-internal package.  Include the installation of this
package in all functional test stages.

Test-tag: daos_racer_parallel daosperf,full_regression

Signed-off-by: Phillip Henderson <phillip.henderson@intel.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -122,7 +122,7 @@ pipeline {
                                     branch: env.BaseBranch,
                                     withSubmodules: true
                         functionalTest inst_repos: daosRepos(),
-                                       inst_rpms: functionalPackages(1, next_version),
+                                       inst_rpms: functionalPackages(1, next_version, "tests-internal"),
```
Comment (brianjmurrell) [inline Jenkinsfile:125]: Is this just a transitional step to baking this new test name right into `pipeline-lib`?

**Review by brianjmurrell — COMMENTED** [review]
As you have noticed, PRs of weekly-testing* branches don't actually run anything.  Making PRs of weekly-testing* branches is not really a workflow that has been put into place so that will probably require some pipeline-lib work to happen.  Maybe PRs of weekly-testing could/should run some much smaller test tag set.

We historically have intentionally not run testing on non-timed commits to allow us to push changes to the branch without tying up clusters for +24h.

File: Jenkinsfile (hunk 1 above)
Comment (phender) [reply Jenkinsfile:125]: Yes.

**Comment by phender** [comment]
I've made the same edit in https://github.com/daos-stack/daos/pull/8902 where for some reason the https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8902/13/ PR does execute tests and it does install the `daos-tests-internal` RPM:
```
[2022-05-24T23:38:22.542Z] + wolf-134:environment:405:post_provision_config_nodes(): retry_dnf 360 install daos-2.3.100-9.8167.gf20c780d.el8 daos-client-2.3.100-9.8167.gf20c780d.el8 daos-tests-2.3.100-9.8167.gf20c780d.el8 daos-server-2.3.100-9.8167.gf20c780d.el8 daos-serialize-2.3.100-9.8167.gf20c780d.el8 daos-tests-internal-2.3.100-9.8167.gf20c780d.el8 ndctl fio patchutils ior romio-tests testmpio python3-mpi4py-tests hdf5-mpich-tests hdf5-openmpi-tests hdf5-vol-daos-openmpi-tests hdf5-vol-daos-mpich-tests simul-mpich simul-openmpi MACSio-mpich MACSio-openmpi mpifileutils-mpich
```

**Review by sylviachanoiyee — APPROVED** [review]

**Review by brianjmurrell — APPROVED** [review]

**Pull request merged**